pyvista/utilities/geometric_objects.py

```diff
@@ -1026,7 +1035,7 @@ def Triangle(points=[[0, 0, 0], [1, 0, 0], [0.5, 0.707, 0]]):
     return triangle
 
 
-def Rectangle(points=[[1, 0, 0], [1, 1, 0], [0, 1, 0], [0, 0, 0]]):
+def Rectangle(points=[[1.0, 0.0, 0], [1.0, 1.0, 0], [0.0, 1.0, 0], [0.0, 0.0, 0.0]]):
```

Comment:
I'd prefer having `points=None` here too and setting the default in the function. It would both be more readable and avoid a mutable default argument.

Reply:
Agreed.  Will fix in a separate PR.

Reply:
```suggestion
def Rectangle(points=None):
```